app/models/widget.js

```diff
@@ -36,6 +36,10 @@ export default Model.extend({
         inverse: 'widgets'
     }),
 
+    elId: Ember.computed(function() {
+        return this.get("widgetType") + this.get("id");
+    }),
+
```

Comment:
for unique identification in the submission form

Reply:
elId is a super ambiguous variable name. rename or add a comment?

Reply:
Yeah that's probably not a bad idea. elementId is better?